posthog/hogql/database/schema/events.py

```diff
@@ -63,6 +63,7 @@ class EventsTable(Table):
         "distinct_id": StringDatabaseField(name="distinct_id"),
         "elements_chain": StringDatabaseField(name="elements_chain"),
         "created_at": DateTimeDatabaseField(name="created_at"),
+        "$session_id": StringDatabaseField(name="$session_id"),
```

Comment:
This is actually a materialized field. I think accessing `properties.$session_id` will direct it to access the field with this name. However fine to add it explicitly too.